spec/footer_spec.rb

```diff
@@ -8,6 +10,8 @@
     @header.click_my_account
     @logon.login('akratz@mcfadyen.com', 'Admin123') # info provisória
```

Comment:
remove logon call as it is not used on the current way to access site. Keeping this call will fail execution